### DISCUSSION TITLE
Traduction of Export CSV Action

### DISCUSSION THREAD

**Discussion by ValentinGaudin** [laravel/nova-issues#5746] [Ideas & Feature Requests]
Hello, it's just a little thing, but maybe you can add the key of traduction for **ExportAsCSV** action : 

```php
    /**
     * Get the displayable name of the action.
     *
     * @return string
     */
    public function name()
    {
        return $this->name = __('Export As CSV');
    }
```

I know we can change in the __construct() method of the ExportAsCSV Action, but the action have already a good name, just had the traduction key. 

Thanks a lot.

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
